src/test/java/org/broadinstitute/hellbender/utils/read/ReadUtilsUnitTest.java

```diff
@@ -7,11 +7,11 @@
 import htsjdk.samtools.reference.IndexedFastaSequenceFile;
```

Comment:
This file might have conflicts after https://github.com/broadinstitute/gatk/pull/3475

Reply:
Already addressed. Done.

Thanks for these comments!